sites/all/modules/reol_app_feeds/src/Feed/FrontPageFeed.php

```diff
@@ -194,6 +196,11 @@ protected function getVideos(array $paragraphIds) {
     ], $paragraphIds);
 
     foreach ($paragraphs as $paragraph) {
+      // Don't include video bundles in Front page feed v2.
+      if (self::class === get_class($this)
```

Comment:
Will `$this` not always be of the type `self:class` or is there some inheritance in play here?  Where this is some type of parent class?

Reply:
Yes, inheritance is the key here. I've updated the comment to make this more clear. I hope.